lexxpluss_apps/src/can_controller.cpp

```diff
@@ -382,7 +382,7 @@ class can_controller_impl {
     const device *dev{nullptr};
     char version_powerboard[32]{""};
     bool heartbeat_timeout{true}, enable_lockdown{true};
-    static constexpr char version[]{"2.7.1"};
+    static constexpr char version[]{"3.0.0"};
```

Comment:
MainBoard/PowerBoard -> SensorControlBoard/PowerDistributeBoardに変わるのでバージョンを継続するのか、0.0.0からスタートするのはか一度考えても良いと思いました。
( この先、MainBoardのファームウェアのメンテナンスもあると思います。 )

I thought it would be good to think once whether to continue the version or start from 0.0.0 as it changes from MainBoard/PowerBoard -> SensorControlBoard/PowerDistributeBoard.
( I think there will be maintenance of MainBoard firmware in the future. )

Reply:
ありがとうございます。ハードの構成も変わったので、V1.0.0からスタートしようと思います。V0.0.0と悩みましたが、MainBoard FirmwareはV0.9はあったが、V1.0.0がスタートのようなので、それにならってV1.0.0スタートとしました。